simtools/model/telescope_model.py

```diff
@@ -34,51 +34,51 @@ class TelescopeModel:
         North or South.
     name: str
         Telescope name for the base set of parameters (e.g., LST-1, ...).
-    modelVersion: str
+    model_version: str
         Version of the model (e.g., prod5).
     label: str
         Instance label.
     mirrors: Mirrors
         Mirrors object created from the mirror list of the model.
     camera: Camera
         Camera object created from the camera config file of the model.
-    referenceData: Reference data
+    reference_data: Reference data
         Dictionary with reference data parameters (e.g., NSB reference value)
-    extraLabel: str
+    extra_label: str
         Extra label to be used in case of multiple telescope configurations (e.g., by ArrayModel).
 
     Methods
     -------
-    fromConfigFile(configFileName, telescopeModelName, label=None)
-        Create a TelescopeModel from a sim_telarray cfg file.
-    setExtraLabel(extraLabel)
+    from_config_file(config_file_name, telescope_model_name, label=None)
+        Create a Telescope_model from a sim_telarray cfg file.
```

Comment:
Considering we are referring to the class here, I think this should be "Creates a TelescopeModel object", no?

Reply:
You are right, changed back.